tests/commands/hook_impl_test.py

```diff
@@ -244,7 +246,8 @@ def test_pushing_orphan_branch(push_example):
         ns = hook_impl._run_ns('pre-push', False, args, stdin)
 
     assert ns is not None
-    assert ns.all_files is True
```

Comment:
I'm curious about this change as well -- what's going on here?

Reply:
Still curious about this change! Would be great to have all the behavior changes from upstream documented somewhere.